File: WebApplication1/WebApplication1/Dtos/GetFormattedResultDto.cs
namespace WebApplication1.Dtos
{
    public class GetFormattedResultDto
    {
        public int Id { get; set; }
        public int CandidateId { get; set; }
        public string CandidateFirstName { get; set; } 
        public string CandidateLastName { get; set; }
        public int StateId { get; set; }
        public int Votes { get; set; }
        public int AllCandidatesVotes { get; set; }
        public bool ErrorFlag { get; set; }

        public int PercentageOfVotes { get; set; }
    }
}

File: PresidentElectionAngularUI/src/app/models/GetFormattedResultDto.ts
export class GetFormattedResultDto
{
     Id: number;
     CandidateId: number;
     CandidateFirstName: string; 
     CandidateLastName: string;
     StateId: number;
     Votes: number;
     AllCandidatesVotes: number;
     ErrorFlag: boolean;
     PercentageOfVotes: number;
}